lib/factory_bot_profiler/frame.rb

```diff
@@ -1,19 +1,35 @@
 module FactoryBotProfiler
   class Frame
-    attr_accessor :name, :start, :duration, :child_time
+    attr_accessor :name, :start, :duration, :child_time, :child_count
 
     def initialize(name)
       @name = name
       @start = Time.now
-      @child_time = 0
+      @child_time = Hash.new { 0 }
+      @child_count = Hash.new { 0 }
     end
 
     def observe_child(frame)
-      @child_time += frame.duration
+      @child_count[frame.name] += 1
+
+      @child_time[frame.name] += frame.self_time
+      frame.child_time.each do |factory_name, time|
```

Comment:
Ah, neato! I was wondering if you were going to separate these out. I like it 👍🏻